filmfest/settings/base.py

```diff
@@ -61,6 +61,8 @@
     'events',
     'results',
     'submissions',
+
+    'partners'
```

Comment:
could you please move this app between `events` and `results`? (don't forget to add a comma